# Docking Force Measurement Parameter File
# ---------------------------------------------------
# Define the connection between load cell 
# amplifier and the raspberry pi

force_sens_00:
  pad_nr: 0
  clk_pin: 23
  dat_pin: 21
  rate: 100
  avg_window: 1
  scale_ratio: 215.9353

force_sens_01:
  pad_nr: 1
  clk_pin: 40
  dat_pin: 35
  rate: 100
  avg_window: 1
  scale_ratio: -206.7201

